pyflask/apis/neuroconv.py

```diff
@@ -122,6 +122,71 @@ def post(self):
                 neuroconv_api.abort(500, str(e))
 
 
+@neuroconv_api.route("/inspect_file")
+class InspectNWBFile(Resource):
+    @neuroconv_api.doc(responses={200: "Success", 400: "Bad Request", 500: "Internal server error"})
+    def post(self):
+        try:
+            import json
+            from nwbinspector import inspect_nwbfile
+            from nwbinspector.nwbinspector import InspectorOutputJSONEncoder
+
+            return json.loads(
+                json.dumps(list(inspect_nwbfile(**neuroconv_api.payload)), cls=InspectorOutputJSONEncoder)
```

Comment:
```suggestion
                json.dumps(
                    list(
                        inspect_nwbfile(
                            ignore=["check_description"],
                            **neuroconv_api.payload
                        )
                    ),
                    cls=InspectorOutputJSONEncoder
                )
```

Suggest suppressing this until we expose metadata control to the users